### PR TITLE
Automate publishing python packages

### PR DESCRIPTION
**Pull request by jashparekh**
## Description

- Automate publishing of wheels to pypi when a new tag is published
- Update classifiers and remove support for python versions that have reached end of life and add classifiers for newer python versions

### CODE REVIEW

**Review by utkbansal — COMMENTED** [review]
Does `invoke release` not work?

**Comment by utkbansal** [comment]
Also, is there a good reason to use `pipenv` here? Can we please stick to tools we already use at other places? This is just confusing.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Changelog
```
Comment (utkbansal) [inline CHANGELOG.md:1]: Should we use this file for the changelog? https://github.com/Instawork/django-admin-fast-search/blob/master/HISTORY.rst

Currently this is the one used for publishing on readthedocs as well.

**Review by utkbansal — COMMENTED** [review]
Another thing to consider (not necessarily in this PR) would be to auto-publish the docs to readthedocs when a new release is made.

File: CHANGELOG.md (hunk 1 above)
Comment (jashparekh) [reply CHANGELOG.md]: Makes sense, made the change

**Comment by jashparekh** [comment]
> Another thing to consider (not necessarily in this PR) would be to auto-publish the docs to readthedocs when a new release is made.

Will add this in a separate PR

File: tasks.py [hunk 2]
```diff
@@ -89,21 +89,25 @@ def lint(c):
     c.run("flake8 django-admin-fast-search tests")
 
 
-@task(help={'bumpsize': 'Bump either for a "feature" or "breaking" change'})
-def release(c, bumpsize=''):
+@task
+def release(c):
     """
     Package and upload a release
```
Comment (simhasrivatsa) [inline tasks.py:95]: do we also create a github release here or only pypi?

Comment (jashparekh) [reply tasks.py:95]: We have to publish a new tag on github, can do a release as well. But essentially publishing a tag is what will trigger the release on PyPi

**Review by utkbansal — APPROVED** [review]

**Pull request merged**